### ISSUE TITLE
Add ablity to export/import ALL color palettes at once

### ISSUE THREAD

**Issue by VinniSauce** [piskelapp/piskel#1078]
I've been using piskel for nearly 4 years now, and one problem that has always plagued the app has been the inability to transfer all of my color palettes from one desktop to another, or to another browser. I could transfer them all one at a time by hand, but i have nearly 100 palettes and it would take several hours to export them all, and then re-import them to the new instance of piskel.

**Comment by ayushmxn** [comment]
<img width="655" alt="image" src="https://user-images.githubusercontent.com/14110965/195213435-f98861c4-8f79-47c6-935d-4de3d7e8565f.png">

@SABITRON I am able to export colour palettes through the Edit Palette dialog one by one as you mentioned above. To understand, you want an `Export All Palettes` option?

**Comment by SABITRON** [comment]
Yes. Export all palettes and Import all palettes as options in some sort of menu, either saving it as a zip folder of palette files or some other kind of file.

**Comment by ayushmxn** [comment]
https://github.com/pahuch/piskel
I'm maintaining a fork of Piskel. Would you be interested in creating a pull request? This seems like a simple feature to add in.
